core/src/main/java/io/fabric8/maven/core/util/ProcessUtil.java

```diff
@@ -123,6 +127,14 @@ protected static void processOutput(Logger log, InputStream inputStream, Functio
     public static File findExecutable(Logger log, String name) {
         List<File> pathDirectories = getPathDirectories(log);
         for (File directory : pathDirectories) {
+            if( isWindows() ) {
```

Comment:
I would prefer to search first on the exact name and then with the extra extensions.